docs/src/development/local/ddev.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Use DDEV for local development
+sidebarTitle: DDEV
+description: Use DDEV to set up local development environments.
+weight: 1
+---
+
+[DDEV](https://ddev.readthedocs.io/en/stable/) is an open-source tool to get local PHP development environments.
+It allows you use Docker in your workflows while maintaining a GitOps workflow.
+You get fully containerized environments to run everything locally
+without having to install tools (such as the Platform.sh CLI) on your machine.
+
+This guide assumes you have a project already running with Platform.sh and you have the code on your computer.
+If you're starting from scratch, first [create a project from a PHP template]({{% create-project-link template=true %}}).
+
+## Before you begin
+
+Make sure your computer meets the [system requirements for DDEV](https://ddev.readthedocs.io/en/stable/#system-requirements).
+
+## 1. Install DDEV
+
+To install DDEV, follow the [DDEV documentation for your operating system](https://ddev.readthedocs.io/en/stable/#installation).
+
+This installs DDEV and `ddev`, its command-line interface (CLI).
+For more information on `ddev`, run `ddev help`.
+
+## 2. Add DDEV configuration to your project
+
+Get basic configuration set up for your project by running the following command:
+
+```bash
+ddev config
+```
+
+If you started with a Drupal template, your repository already had DDEV configuration files.
+Otherwise, you have to answer a couple of questions about what your repository is
+so the correct files are added.
+
+Test the configuration by running the following command:
+
+```bash
+ddev start
+```
+
+Your project should start running, though without any of the data from your Platform.sh environment.
+
+## 3. Add an API token
+
+To connect DDEV with your Platform.sh account, use a Platform.sh API token.
+
+1. [Create an API token](./../cli/api-tokens.md#get-a-token) in the console.
+2. Add the token to your global DDEV configuration:
+
+   ```yaml {location="~/.ddev/global_config.yaml"}
+   web_environment:
+       - PLATFORMSH_CLI_TOKEN=<API_TOKEN>
+   ```
+
+   Note that this isn't the configuration file in your project.
+3. Get DDEV to recognize the token by restarting DDEV:
+
+   ```bash
+   ddev restart
+   ```
+
+Now you can run `ddev exec platform <command>` from your computer without needing to install the Platform.sh CLI.
```

Comment:
A platform DDEV add-on could be a great thing. `ddev get platformsh/platform-setup` could add the .ddev/providers/platform.sh, might even be able to figure out what the env values should be. It could also install a custom command for `ddev platform`.  https://github.com/drud/ddev-addon-template